tests/test_algorithm.py

```diff
@@ -1595,13 +1625,15 @@ def tearDownClass(cls):
         cls.tempdir.cleanup()
 
     def test_auto_close_future(self):
-        self.algo = TestAlgorithm(sid=1, amount=1, order_count=1,
-                                  instant_fill=True, commission=PerShare(0),
```

Comment:
Did this affect the results? I don't see a change to the asserts, but would have assumed the fills would have changed resulting positions etc.